doc/api/test.md

```diff
@@ -2224,7 +2224,7 @@ added:
   - v18.19.0
```

Comment:
Does an entry need to be added here?

Reply:
I think this goes automatically added by the CI

Reply:
done!